conf/database-connection-admin.env

```diff
@@ -1,3 +1,5 @@
 POSTGRESQL_DATABASE=vulnerability
 POSTGRESQL_HOST=database
 POSTGRESQL_PORT=5432
```

Comment:
The other two configuration files habe POSTGRESQL_PORT=15432, shouldn't that value be there as well instead of 5432?

Reply:
I see that in dockerfile there's mapping from 15432 port to 5432 port so this is most likely not an issue, but still asking just to make extra sure.